charts/newrelic-infrastructure/values.yaml

```diff
@@ -449,7 +449,7 @@ customAttributes: {}
 # @default -- See `values.yaml`
 serviceAccount:
   # -- Whether the chart should automatically create the ServiceAccount objects required to run.
-  create: true
```

Comment:
yes , i also added some test , that i will re-work since i just discovered this 
https://github.com/newrelic/nri-kubernetes/pull/513#issuecomment-1183370327